tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_lockscreen_notification.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.wait import Wait
+
+from gaiatest.apps.lockscreen.app import LockScreen
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+
+
+class TestLockScreen(GaiaImageCompareTestCase):
+
+    _notification_title = 'TestNotificationBar_TITLE'
+    _notification_body = 'TestNotificationBar_BODY'
+    _seconds_since_epoch = 1357043430
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+
+        self.data_layer.set_time(self._seconds_since_epoch * 1000)
+        self.data_layer.set_setting('time.timezone', 'Atlantic/Reykjavik')
+        # this time we need it locked!
+        self.device.lock()
+
+    def test_lockscreen_notification(self):
+
+        Wait(self.marionette, timeout=30).until(lambda m: self.device.has_mobile_connection)
```

Comment:
I see, perhaps we can change some setting to make this consistent, such as disabling cell network? Alternatively, please add a comment.